natlas-server/app/static/util/version-check.ts

```diff
@@ -12,9 +13,10 @@ class UpdateCheckResult {
     }
 }
 
-// Not ideal. We should pack this into the bundle instead
+declare const NATLAS_VERSION: string;
+
 export function thisVersion(): string {
-    return document.getElementById('natlasVersion').innerText;
+    return NATLAS_VERSION;
```

Comment:
What are your thoughts on using this embedded version info to display in the global footer? If someone isn't using sentry then they won't see this better versioning information and instead only see the not-so-good embedded version from the jinja template. It'd be nice if we could just embed this information into the python application itself as well but I know that's beyond the scope of this PR and a harder problem to solve.

Reply:
This would be a good idea. That element comes from the Python Config file right? We could embed the same logic into Python to satisfy that.

Reply:
https://github.com/natlas/natlas/blob/bb9f2d9a54d4ed7c6e765604e4d82e2ec402b85b/natlas-server/app/templates/includes/footer.html#L12 This is the line where the actual version info is being rendered, but it does ultimately come from the `config.py` file.

I don't think we can easily replace the `app.config['NATLAS_VERSION']` across the entire application at this time because the agent api compares the agent version with the server version and rejects if there's a mismatch (because the client-server protocol is not independently versioned at this time). For now I think just having the javascript overwrite (or append) to the version information in the footer is the least intrusive way until we can fix the transport communication problem. (protobufs? 😏)

Reply:
I'll leave it up to you whether or not you want to do javascript augmentation of this value or if you'd prefer to just hold out until we can do it it uniformly (once the transport between agent-server is independently versioned then it's easy)

Reply:
I'm going to merge as is because it's broken right now, but I don't think we're going to have an issue with the versioning since we want the agent to match the server anyway. We just need a common version algorithm across both